README.md

```diff
@@ -24,15 +24,15 @@ This platform is built upon three principles:
 - Framework: [Next.js](https://nextjs.org)
   - [CSS Modules](https://nextjs.org/docs/basic-features/built-in-css-support)
   - [TypeScript](https://nextjs.org/docs/basic-features/typescript)
-- CMS: [DatoCMS](https://www.datocms.com)
+- CMS: [Agility CMS](https://www.agilitycms.com)
```

Comment:
Could you follow the other PRs example for the README changes? Feel free to copy paste where's there.